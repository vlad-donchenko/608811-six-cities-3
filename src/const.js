const ZOOM = 12;

const MAX_NEIGHBOURHOOD_OFFERS = 3;

const monthMap = {
  "0": `January`,
  "1": `February`,
  "2": `March`,
  "3": `April`,
  "4": `May`,
  "5": `June`,
  "6": `July`,
  "7": `August`,
  "8": `September`,
  "9": `October`,
  "10": `November`,
  "11": `December`,
};

const OfferPrefix = {
  MAIN: `cities__place-card`,
  DETAILS: `near-places__card`,
};

export {ZOOM, MAX_NEIGHBOURHOOD_OFFERS, OfferPrefix, monthMap};
