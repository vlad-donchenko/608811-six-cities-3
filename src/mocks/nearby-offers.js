const nearbyOffers = [
  {
    bedrooms: 5,
    city: {
      location: {
        latitude: 52.38333,
        longitude: 4.9,
        zoom: 10
      },
      name: `Amsterdam`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century`,
    goods: [`Wi-Fi`, `Heating`, `Kitchen`, `Fridge`, `Washing machine`, `Baby seat`],
    host: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 50,
      isPro: true,
      name: `Angelina`
    },
    id: 50,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: true,
    location: {
      latitude: 52.398298,
      longitude: 4.848328,
      zoom: 8
    },
    maxAdults: 3,
    previewImage: `apartment-01.jpg`,
    price: 424,
    rating: 4,
    title: `Beautiful & luxurious apartment at great location - Nearby`,
    type: `Private room`
  },
  {
    bedrooms: 5,
    city: {
      location: {
        latitude: 52.38333,
        longitude: 4.9,
        zoom: 10
      },
      name: `Amsterdam`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century`,
    goods: [`Wi-Fi`, `Heating`, `Kitchen`, `Fridge`, `Washing machine`, `Baby seat`],
    host: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 51,
      isPro: true,
      name: `Angelina`
    },
    id: 51,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: false,
    location: {
      latitude: 52.397288,
      longitude: 4.852012,
      zoom: 8
    },
    maxAdults: 3,
    previewImage: `apartment-01.jpg`,
    price: 124,
    rating: 2,
    title: `Beautiful & luxurious apartment at great location - Nearby`,
    type: `Private room`
  },
  {
    bedrooms: 5,
    city: {
      location: {
        latitude: 52.38333,
        longitude: 4.9,
        zoom: 10
      },
      name: `Amsterdam`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century`,
    goods: [`Wi-Fi`, `Heating`, `Kitchen`, `Fridge`, `Washing machine`, `Baby seat`],
    host: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 52,
      isPro: true,
      name: `Angelina`
    },
    id: 52,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: true,
    isPremium: true,
    location: {
      latitude: 52.395049,
      longitude: 4.854962,
      zoom: 8
    },
    maxAdults: 3,
    previewImage: `apartment-02.jpg`,
    price: 924,
    rating: 2,
    title: `Beautiful & luxurious apartment at great location - Nearby`,
    type: `Private room`
  },
  {
    bedrooms: 3,
    city: {
      location: {
        latitude: 48.866411,
        longitude: 2.331896,
        zoom: 15
      },
      name: `Paris`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Wi-Fi`, `Kitchen`, `Towels`, `Baby seat`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 53,
      isPro: true,
      name: `Max`
    },
    id: 53,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: false,
    location: {
      latitude: 48.869508,
      longitude: 2.327385,
      zoom: 8
    },
    maxAdults: 4,
    previewImage: `apartment-02.jpg`,
    price: 1100,
    rating: 1,
    title: `Sofitel Paris Le Faubourg - Nearby`,
    type: `Apartment`
  },
  {
    bedrooms: 3,
    city: {
      location: {
        latitude: 48.866411,
        longitude: 2.331896,
        zoom: 15
      },
      name: `Paris`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Wi-Fi`, `Kitchen`, `Towels`, `Baby seat`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 54,
      isPro: true,
      name: `Max`
    },
    id: 54,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: true,
    isPremium: true,
    location: {
      latitude: 48.870955,
      longitude: 2.330679,
      zoom: 8
    },
    maxAdults: 4,
    previewImage: `apartment-02.jpg`,
    price: 100,
    rating: 5,
    title: `Sofitel Paris Le Faubourg - Nearby`,
    type: `Apartment`
  },
  {
    bedrooms: 3,
    city: {
      location: {
        latitude: 48.866411,
        longitude: 2.331896,
        zoom: 15
      },
      name: `Paris`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Wi-Fi`, `Kitchen`, `Towels`, `Baby seat`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 55,
      isPro: true,
      name: `Max`
    },
    id: 55,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: true,
    location: {
      latitude: 48.869918,
      longitude: 2.327986,
      zoom: 8
    },
    maxAdults: 4,
    previewImage: `apartment-03.jpg`,
    price: 190,
    rating: 3,
    title: `Sofitel Paris Le Faubourg - Nearby`,
    type: `Apartment`
  }
];

export default nearbyOffers;
