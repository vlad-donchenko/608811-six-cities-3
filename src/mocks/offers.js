const offers = [
  {
    bedrooms: 5,
    city: {
      location: {
        latitude: 52.38333,
        longitude: 4.9,
        zoom: 10
      },
      name: `Amsterdam`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century`,
    goods: [`Wi-Fi`, `Heating`, `Kitchen`, `Fridge`, `Washing machine`, `Baby seat`],
    host: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 0,
      isPro: true,
      name: `Angelina`
    },
    id: 0,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: true,
    isPremium: true,
    location: {
      latitude: 52.3909553943508,
      longitude: 4.85309666406198,
      zoom: 8
    },
    maxAdults: 3,
    previewImage: `apartment-01.jpg`,
    price: 494,
    rating: 5,
    title: `Beautiful & luxurious apartment at great location`,
    type: `Private room`
  },
  {
    bedrooms: 1,
    city: {
      location: {
        latitude: 52.38333,
        longitude: 4.9,
        zoom: 10
      },
      name: `Amsterdam`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful. A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century`,
    goods: [`Heating`, `Coffee machine`, `Dishwasher`, `Towels`, `Baby seat`, `Cabel TV`],
    host: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 1,
      isPro: false,
      name: `Max`
    },
    id: 1,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: false,
    location: {
      latitude: 52.369553943508,
      longitude: 4.85309666406198,
      zoom: 8
    },
    maxAdults: 5,
    previewImage: `apartment-01.jpg`,
    price: 264,
    rating: 1,
    title: `Wood and stone place`,
    type: `Private room`
  },
  {
    bedrooms: 4,
    city: {
      location: {
        latitude: 52.38333,
        longitude: 4.9,
        zoom: 10
      },
      name: `Amsterdam`
    },
    description: `An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful. A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century`,
    goods: [`Fridge`, `Dishwasher`, `Towels`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 2,
      isPro: false,
      name: `Sam`
    },
    id: 2,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: true,
    isPremium: false,
    location: {
      latitude: 52.3909553943508,
      longitude: 4.929309666406198,
      zoom: 8
    },
    maxAdults: 2,
    previewImage: `apartment-02.jpg`,
    price: 418,
    rating: 5,
    title: `Canal View Prinsengracht`,
    type: `Private room`
  },
  {
    bedrooms: 7,
    city: {
      location: {
        latitude: 52.38333,
        longitude: 4.9,
        zoom: 10
      },
      name: `Amsterdam`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century. A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam. The building is green and from 18th century`,
    goods: [`Heating`, `Kitchen`, `Fridge`, `Washing machine`, `Dishwasher`, `Baby seat`, `Cabel TV`],
    host: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 3,
      isPro: true,
      name: `Nick`
    },
    id: 3,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: false,
    location: {
      latitude: 52.3809553943508,
      longitude: 4.939309666406198,
      zoom: 8
    },
    maxAdults: 7,
    previewImage: `apartment-03.jpg`,
    price: 358,
    rating: 5,
    title: `Nice, cozy, warm big bed apartment`,
    type: `Private room`
  },
  {
    bedrooms: 2,
    city: {
      location: {
        latitude: 52.38333,
        longitude: 4.9,
        zoom: 10
      },
      name: `Amsterdam`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Wi-Fi`, `Kitchen`, `Coffee machine`, `Towels`, `Baby seat`],
    host: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 4,
      isPro: true,
      name: `Jane`
    },
    id: 4,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: false,
    location: {
      latitude: 52.3709553943508,
      longitude: 4.979309666406198,
      zoom: 8
    },
    maxAdults: 4,
    previewImage: `apartment-02.jpg`,
    price: 402,
    rating: 5,
    title: `Wood and stone place`,
    type: `Apartment`
  },
  {
    bedrooms: 4,
    city: {
      location: {
        latitude: 48.866411,
        longitude: 2.331896,
        zoom: 15
      },
      name: `Paris`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Wi-Fi`, `Kitchen`, `Towels`, `Baby seat`],
    host: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 5,
      isPro: true,
      name: `Jane`
    },
    id: 5,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: false,
    location: {
      latitude: 48.866347,
      longitude: 2.332926,
      zoom: 8
    },
    maxAdults: 4,
    previewImage: `apartment-02.jpg`,
    price: 402,
    rating: 5,
    title: `Le Roch Hotel & Spa`,
    type: `Apartment`
  },
  {
    bedrooms: 3,
    city: {
      location: {
        latitude: 48.866411,
        longitude: 2.331896,
        zoom: 15
      },
      name: `Paris`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Wi-Fi`, `Kitchen`, `Towels`, `Baby seat`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 6,
      isPro: true,
      name: `Max`
    },
    id: 6,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: true,
    isPremium: true,
    location: {
      latitude: 48.864967,
      longitude: 2.329262,
      zoom: 8
    },
    maxAdults: 10,
    previewImage: `apartment-03.jpg`,
    price: 800,
    rating: 4,
    title: `Hotel Brighton`,
    type: `Apartment`
  },
  {
    bedrooms: 4,
    city: {
      location: {
        latitude: 50.946639,
        longitude: 6.961174,
        zoom: 13
      },
      name: `Cologne`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Heating`, `Dishwasher`, `Towels`, `Baby seat`, `Cabel TV`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 7,
      isPro: false,
      name: `Sam`
    },
    id: 7,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: true,
    isPremium: false,
    location: {
      latitude: 48.864413,
      longitude: 2.330791,
      zoom: 8
    },
    maxAdults: 8,
    previewImage: `apartment-02.jpg`,
    price: 700,
    rating: 4,
    title: `DJH Youth Hostel Cologne Pathpoint`,
    type: `Apartment`
  },
  {
    bedrooms: 4,
    city: {
      location: {
        latitude: 50.946639,
        longitude: 6.961174,
        zoom: 13
      },
      name: `Cologne`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Heating`, `Dishwasher`, `Towels`, `Baby seat`, `Cabel TV`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 8,
      isPro: true,
      name: `Sam`
    },
    id: 8,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: true,
    isPremium: true,
    location: {
      latitude: 50.945373,
      longitude: 6.958623,
      zoom: 8
    },
    maxAdults: 8,
    previewImage: `apartment-01.jpg`,
    price: 900,
    rating: 4,
    title: `Hotel Domspitzen`,
    type: `Apartment`
  },
  {
    bedrooms: 4,
    city: {
      location: {
        latitude: 50.946639,
        longitude: 6.961174,
        zoom: 13
      },
      name: `Cologne`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Heating`, `Dishwasher`, `Towels`, `Baby seat`, `Cabel TV`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 9,
      isPro: false,
      name: `Sam`
    },
    id: 9,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: false,
    location: {
      latitude: 50.945900,
      longitude: 6.957892,
      zoom: 8
    },
    maxAdults: 8,
    previewImage: `apartment-01.jpg`,
    price: 1100,
    rating: 4,
    title: `Hotel Merian`,
    type: `Apartment`
  },
  {
    bedrooms: 4,
    city: {
      location: {
        latitude: 50.946639,
        longitude: 6.961174,
        zoom: 13
      },
      name: `Cologne`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Heating`, `Dishwasher`, `Towels`, `Baby seat`, `Cabel TV`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 11,
      isPro: true,
      name: `Max`
    },
    id: 11,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: true,
    location: {
      latitude: 50.945450,
      longitude: 6.955685,
      zoom: 8
    },
    maxAdults: 8,
    previewImage: `apartment-01.jpg`,
    price: 100,
    rating: 4,
    title: `a&o Hostel Köln Hauptbahnhof`,
    type: `Apartment`
  },
  {
    bedrooms: 4,
    city: {
      location: {
        latitude: 50.848587,
        longitude: 4.342344,
        zoom: 10
      },
      name: `Brussels`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Heating`, `Dishwasher`, `Towels`, `Baby seat`, `Cabel TV`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 12,
      isPro: true,
      name: `Max`
    },
    id: 12,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: true,
    location: {
      latitude: 50.833595,
      longitude: 4.354622,
      zoom: 8
    },
    maxAdults: 8,
    previewImage: `apartment-01.jpg`,
    price: 570,
    rating: 4,
    title: `a&o Hostel Köln Hauptbahnhof`,
    type: `Apartment`
  },
  {
    bedrooms: 4,
    city: {
      location: {
        latitude: 50.848587,
        longitude: 4.342344,
        zoom: 10
      },
      name: `Brussels`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Heating`, `Dishwasher`, `Towels`, `Baby seat`, `Cabel TV`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 13,
      isPro: true,
      name: `Max`
    },
    id: 13,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: true,
    location: {
      latitude: 50.848587,
      longitude: 4.342344,
      zoom: 8
    },
    maxAdults: 8,
    previewImage: `apartment-03.jpg`,
    price: 570,
    rating: 4,
    title: `Citadines Toison d'Or Brussels (Apart Hotel Brussels)`,
    type: `Apartment`
  },
  {
    bedrooms: 4,
    city: {
      location: {
        latitude: 50.946639,
        longitude: 6.961174,
        zoom: 10
      },
      name: `Brussels`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Heating`, `Dishwasher`, `Towels`, `Baby seat`, `Cabel TV`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 14,
      isPro: false,
      name: `Max`
    },
    id: 14,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: true,
    isPremium: false,
    location: {
      latitude: 50.833025,
      longitude: 4.356767,
      zoom: 8
    },
    maxAdults: 8,
    previewImage: `room.jpg`,
    price: 270,
    rating: 5,
    title: `Hotel NH Brussels Louise`,
    type: `Apartment`
  },
  {
    bedrooms: 4,
    city: {
      location: {
        latitude: 53.551059,
        longitude: 9.977448,
        zoom: 13
      },
      name: `Hamburg`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Heating`, `Dishwasher`, `Towels`, `Baby seat`, `Cabel TV`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 15,
      isPro: false,
      name: `Max`
    },
    id: 15,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: true,
    location: {
      latitude: 53.549422,
      longitude: 9.967592,
      zoom: 8
    },
    maxAdults: 8,
    previewImage: `room.jpg`,
    price: 3660,
    rating: 5,
    title: `ARCOTEL Onyx Hamburg`,
    type: `Apartment`
  },
  {
    bedrooms: 4,
    city: {
      location: {
        latitude: 53.551059,
        longitude: 9.977448,
        zoom: 13
      },
      name: `Hamburg`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Heating`, `Dishwasher`, `Towels`, `Baby seat`, `Cabel TV`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 16,
      isPro: false,
      name: `Max`
    },
    id: 16,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: false,
    location: {
      latitude: 53.551360,
      longitude: 9.966766,
      zoom: 8
    },
    maxAdults: 8,
    previewImage: `apartment-02.jpg`,
    price: 1660,
    rating: 5,
    title: `Holiday Inn Express Hamburg-St. Pauli Messe`,
    type: `Apartment`
  },
  {
    bedrooms: 4,
    city: {
      location: {
        latitude: 51.227738,
        longitude: 6.793251,
        zoom: 10
      },
      name: `Dusseldorf`
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    goods: [`Heating`, `Dishwasher`, `Towels`, `Baby seat`, `Cabel TV`],
    host: {
      avatarUrl: `avatar-max.jpg`,
      id: 17,
      isPro: false,
      name: `Max`
    },
    id: 17,
    images: [`room.jpg`, `apartment-02.jpg`, `apartment-03.jpg`, `apartment-01.jpg`, `apartment-01.jpg`],
    isFavorite: false,
    isPremium: false,
    location: {
      latitude: 51.220447,
      longitude: 6.789114,
      zoom: 8
    },
    maxAdults: 8,
    previewImage: `apartment-02.jpg`,
    price: 160,
    rating: 4,
    title: `Hotel Düsseldorf City by Tulip Inn`,
    type: `Apartment`
  }
];

export {offers};
