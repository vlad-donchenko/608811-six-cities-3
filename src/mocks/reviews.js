const reviews = [
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-05-08T14:13:56.569Z`,
    id: 1,
    rating: 4,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 1,
      isPro: false,
      name: `Max`
    }
  },
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-06-08T14:13:56.569Z`,
    id: 2,
    rating: 5,
    user: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 2,
      isPro: true,
      name: `Angelina`
    }
  },
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-09-08T14:13:56.569Z`,
    id: 3,
    rating: 1,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 3,
      isPro: false,
      name: `Max`
    }
  },
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-10-08T14:13:56.569Z`,
    id: 4,
    rating: 5,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 4,
      isPro: false,
      name: `Angelina`
    }
  },
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-10-08T14:13:56.569Z`,
    id: 5,
    rating: 2,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 3,
      isPro: true,
      name: `Max`
    }
  }
];

export {reviews};
