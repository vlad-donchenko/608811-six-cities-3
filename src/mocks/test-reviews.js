const reviews = [
  {
    comment: `Lorem ipsum dolor sit amet, consectetur adipisicing elit.`,
    date: `2019-08-08T14:13:56.569Z`,
    id: 1,
    rating: 5,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 1,
      isPro: true,
      name: `Max`
    }
  },
  {
    comment: `Lorem ipsum dolor sit amet, consectetur adipisicing elit.`,
    date: `2019-06-03T14:13:56.569Z`,
    id: 2,
    rating: 4,
    user: {
      avatarUrl: `avatar-angelina.jpg`,
      id: 2,
      isPro: false,
      name: `Angelina`
    }
  },
  {
    comment: `Lorem ipsum dolor sit amet, consectetur adipisicing elit.`,
    date: `2019-06-01T14:13:56.569Z`,
    id: 3,
    rating: 2,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 3,
      isPro: false,
      name: `Max`
    }
  },
  {
    comment: `Lorem ipsum dolor sit amet, consectetur adipisicing elit.`,
    date: `2019-11-08T14:13:56.569Z`,
    id: 4,
    rating: 2,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 4,
      isPro: false,
      name: `Angelina`
    }
  },
  {
    comment: `Lorem ipsum dolor sit amet, consectetur adipisicing elit.`,
    date: `2019-01-08T14:13:56.569Z`,
    id: 5,
    rating: 4,
    user: {
      avatarUrl: `avatar-max.jpg`,
      id: 3,
      isPro: false,
      name: `Max`
    }
  }
];

export {reviews};
