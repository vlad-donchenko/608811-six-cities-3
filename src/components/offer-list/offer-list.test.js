import React from "react";
import renderer from "react-test-renderer";
import OfferList from "./offer-list.jsx";
import {offers} from "../../mocks/test-offer";
import {OfferPrefix} from "../../const";


it(`Should render <OfferList/>`, () => {
  const tree = renderer.create(<OfferList additionalClass={OfferPrefix.CITIES_PLACE_CARD} renderOffer={() => {}} offers={offers} onTitleClick={() => {
  }}/>).toJSON();

  expect(tree).toMatchSnapshot();
});
