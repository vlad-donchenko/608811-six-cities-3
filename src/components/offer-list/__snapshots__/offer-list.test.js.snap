// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render <OfferList/> 1`] = `
<div
  className="cities__places-list places__list tabs__content"
/>
`;
